deploy/iso/minikube-iso/Dockerfile

```diff
@@ -37,11 +37,22 @@ RUN apt-get update \
 		golang-go \
 		libpcre3-dev \
 		mkisofs \
+		qemu-system-arm \
+		qemu-system-x86 \
+		libconfuse-dev \
 	&& rm -rf /var/lib/apt/lists/*
 
 RUN localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 ENV LANG en_US.utf8
 
+# Build genimage (required by some targets to generate disk images)
```

Comment:
This is supposed to be built by buildroot, and should **not** be needed on the host.

`BR2_PACKAGE_HOST_GENIMAGE`

Reply:
Sounds good.